docs/index.md

```diff
@@ -159,7 +159,11 @@ JWT_AUTH = {
     'JWT_AUTH_HEADER_PREFIX': 'Bearer',
     'JWT_RESPONSE_PAYLOAD_HANDLER':
         'rest_framework_jwt.utils.jwt_create_response_payload',
-    'JWT_AUTH_COOKIE': None
+    'JWT_AUTH_COOKIE': None,
+    'JWT_AUTH_COOKIE_DOMAIN': None,
+    'JWT_AUTH_COOKIE_PATH': None,
```

Comment:
Thank you for spotting this oversight on my part. Yes, the default should mirror `SESSION_COOKIE_PATH` and I have changed it.

FTR, the net effect is the same as `None` because `/` is the default in `set_cookie()`:

https://github.com/django/django/blob/fa6893e9dbb127d70c1376667c700b71bb987fe0/django/http/response.py#L158